.github/workflows/ui-tests.yml

```diff
@@ -40,30 +43,12 @@ jobs:
         browser: ['chrome', 'firefox', 'edge']
 
     runs-on: ubuntu-latest
-    services:
```

Comment:
I did a bit of research and was able to find out that
- We can't reference local Dockerfiles in `services`.
- What most of the people did was they just created a local dockerfile/docker-compose.yml file and ran them using `docker-compose`.